enterprise/templates/enterprise/templatetags/course_modal.html

```diff
@@ -73,6 +73,19 @@ <h1 id="modal-header-text-{{ index }}" class="modal-header-text">{{ course_title
                 </div>
               </li>
             {% endif %}
+            {% if course_duration %}
+              <li>
+                <div class="detail-container">
+                  <div class="detail-title-container">
+                    <span class="icon fa fa-tachometer" aria-hidden="true"></span>
```

Comment:
Should we use a different icon? may be `fa-clock`?

Reply:
Done